src/components/shared/card/card-profile/index.tsx

```diff
@@ -11,17 +13,17 @@ const DEFAULT_SIZE = 16;
 export const CardProfile = ({ nickName, profileImage, size = DEFAULT_SIZE }: CardProfileProps) => {
   return (
     <div className='mt-3 flex items-center gap-1.5'>
-      {profileImage ? (
+      <div
+        className='relative shrink-0 overflow-hidden rounded-full'
+        style={{ width: size, height: size }}
+      >
         <Image
```

Comment:
```suggestion
        <ImageWithFallback
          src={profileImage}
        />
```
<ImageWithFallback /> 컴포넌트는 fallbackSrc 프롭스를 넘기지 않을 경우, 디폴트 값으로 강아지 Profile 사진이 적용됩니다.
즉, 그냥 src={profileImage} 만 적용하시면 OK